docs/book/src/quick-start.md

```diff
@@ -24,60 +34,53 @@ sudo mv /tmp/kubebuilder_2.0.0_${os}_${arch} /usr/local/kubebuilder
 export PATH=$PATH:/usr/local/kubebuilder/bin
 ```
 
-You can also install a KubeBuilder master snapshot from
-`https://go.kubebuilder.io/dl/latest/${os}/${arch}`.
+**NOTE:** Also, you can install a master snapshot from `https://go.kubebuilder.io/dl/latest/${os}/${arch}`.
 
-Install [kustomize](https://sigs.k8s.io/kustomize/docs/INSTALL.md) v3.1.0+
+* Install [kustomize](https://sigs.k8s.io/kustomize/docs/INSTALL.md) v3.1.0+
 
 ## Create a Project
 
-Initialize a new project and Go module for your controllers:
+Create a directory then run the following command inside of it to initialize a new project:
 
 ```bash
 kubebuilder init --domain my.domain
 ```
 
-If you're not in `GOPATH`, you'll need to run `go mod init <modulename>`
+**NOTE:** If you're not in `GOPATH`, you'll need to run `go mod init <modulename>`
 in order to tell kubebuilder and Go the base import path of your module.
 
 ## Create an API
 
-Create a new API group-version called `webapp/v1`, and a kind `Guestbook`
-in that API group-version:
+* Run the following command to create a new API (group/version) as `webapp/v1` and the new Kind(CRD) `Guestbook` on it:
 
 ```bash
 kubebuilder create api --group webapp --version v1 --kind Guestbook
 ```
 
-This will create the files `api/v1/guestbook_types.go` and
-`controller/guestbook_controller.go` for you to edit.
+**NOTE:** This will create the files `api/v1/guestbook_types.go` where the API is defined 
```

Comment:
make this a note block along the same lines as the others (look for `<aside class="note">`)

Reply:
ditto elsewere in your changes

Reply:
I tried to use : 

```
<aside class="notes" data-markdown>
{note}
</aside>
```
and it did not work. I mean, was the same as nothing. I never ever saw it before. How it supose render? Should it add the NOTE?